akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -121,6 +121,9 @@ akka.cluster.sharding {
   # Only used when state-store-mode=ddata
   updating-state-timeout = 5 s
 
+  # Timeout to wait for querying all shards for a given `ShardRegion`.
```

Comment:
This might be a bit obscure for the users' perspective but :+1:

Reply:
Quite true but the parent ticket of this (noted above, and its PR coming) was raised by a user where part of their ask is about making these timeouts configurable. I think it is a good idea.

Reply:
I agree